internal/access/signup.go

```diff
@@ -73,9 +76,22 @@ func Signup(c *gin.Context, name, password, orgName string) (*models.Identity, *
 
 	for _, grant := range grants {
 		if err := data.CreateGrant(db, grant); err != nil {
-			return nil, nil, err
+			return nil, "", fmt.Errorf("create grant on sign-up: %w", err)
 		}
 	}
 
-	return identity, org, nil
+	// grant the user a session on initial sign-up
+	accessKey := &models.AccessKey{
+		IssuedFor:         identity.ID,
+		IssuedForIdentity: identity,
```

Comment:
I think we only need to set `IssusedFor`, not `IssusedForIdentity`. Did you see failures when this was not set? Also not a blocker

Reply:
This part came from copy/pasting the login access key creation, the "IssuedForIdentity" is nil after save when you dont assign it so I assigned it to be safer.